.github/CODEOWNERS

```diff
@@ -0,0 +1,24 @@
+# Each line is a file pattern followed by one or more owners.
+# Owners will be requested for review when someone opens a pull request.
+
+# For more details, see https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
+
+/compose/ @dockertopia @usha-mandya
+
+/compose/install.md @aevesdocker
+
+/desktop/ @aevesdocker @usha-mandya
+
+/docker-hub/ @jerae-duffin
+
+/docker-hub/orgs.md @craig-osterhout
```

Comment:
```suggestion
/docker-hub/orgs.md @craig-osterhout @jerae-duffin 
```

Reply:
Removed this entry. With the more general /docker-hub/ entry, we'll both be notified if this file changes.